server.js

```diff
@@ -1,16 +1,32 @@
 const express = require('express');
 const app = express();
+const morgan = require('morgan')
+
+app.use(morgan('dev'))
+app.use(express.json())
+
+const logBody = (req, res, next) => {
+  console.log(req.body)
+  next()
+}
+
+const fullName = (req, res, next) => {
+  if (!req.body.firstName && !req.body.lastName) return (res.json({message: 'oy div, send me proper data'}))
+  const fullName = `${req.body.firstName} ${req.body.lastName}`
+  req.body.fullName = fullName
```

Comment:
Nice trick :) Have actually never tried that. 
But if the logBody middleware displays the fullname, then I will definitely use that in the future too ;)